kpi-tracker/Jenkinsfile

```
@@ -0,0 +1,41 @@
+@Library('concordium-pipelines') _
+pipeline {
+    agent any
+    environment {
+        image_repo = "concordium/concordium-misc-tools"
```

This will be the docker image name, so it should probably be something like:
```suggestion
        image_repo = "concordium/kpi-tracker"
```